pages/common/cpio.md

```diff
@@ -1,16 +1,15 @@
 # cpio
 
-> Archiving utility.
+> Copies files to and from archives.
```

Comment:
Yes, like I mentioned before, I thought it was a bit weird to mention copy in and copy out. But since you are okay, I will change it.